indico/core/signals/event/designer.py

```diff
@@ -22,3 +22,8 @@
 Called when printing a badge template. Registration form is passed in the 'regform' kwarg.
 kwarg.
 """)
+
+generate_ticket_qr_code = _signals.signal('generate-ticket-qr-code', """
```

Comment:
isn't this signal more suitable in signals/event/registration.py? ticketing is more a registration feature than a designer feature.